test/parser.js

```diff
@@ -0,0 +1,77 @@
+import { expect } from 'chai';
+import fury from 'fury';
+import Parser from '../src/parser';
+
+describe('Parser', () => {
+  let parser;
+
+  before(() => {
+    parser = new Parser({ minim: fury.minim });
+    parser.swagger = {
+      consumes: [],
+      produces: [],
+    };
+  });
+
+  context('content types', () => {
+    it('gathers null response type when no examples or produces', () => {
+      const methodValue = {};
+      const examples = {};
+      const contentTypes = parser.gatherResponseContentTypes(methodValue, examples);
+
+      expect(contentTypes).to.deep.equal([null]);
```

Comment:
I haven't inspected the subsequent code, but an interface using an array containing a `null` to signalize there are no content types looks weird. Wouldn't an empty array be better?

Reply:
These content types are mapped directly into request/response. `null` represents an empty content type and thus simplifies code elsewhere. There is code similar to:

```javascript
responseContentTypes.forEach((responseContentType) => {
  requestContentTypes.forEach((requestContentType) => {

  })
})
```

Where if either content-type is null it just represents a request/response without a content type. Perhaps the `ContentTypes` component of the method name could be clearer. These methods are generating all of the possible request/response pairs and their respective content type.

Reply:
I see. Makes sense!